dev-util/SPIRV-Tools/SPIRV-Tools-9999.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-multilib cmake-utils git-r3
```

Comment:
Why do you need it multilib?

Reply:
my end goal is to add the layers USE to vulkan-loader, which requires spirv-tools. vulkan-loader is multilib and requires binaries from spirv-tools